src/edu/csus/ecs/pc2/ui/EditProblemPane.java

```diff
@@ -248,7 +248,30 @@ public class EditProblemPane extends JPanePlugin {
     
     private EditProblemSandboxPane editProblemSandboxPane;
 
-
+    /**
+     * This enum defines the specification followed by the custom validator
+     */
+    private enum CUSTOM_VALIDATOR_TYPE {
```

Comment:
I think this is a bad name for this enum.  The enum name implies that it is an enumeration of _types of custom validators_; in fact, it is an enumeration of the _types of **interfaces** which a custom validator can use to communicate with PC2_.  I think it should be changed to (something like) `CUSTOM_VALIDATOR_INTERFACE_TYPE`.

Reply:
Will be fixed in the next commit.